bindings/nodejs/lib/types/client/output_builder_params/alias-output-params.ts

```diff
@@ -4,12 +4,27 @@ import { Feature, HexEncodedString } from '../..';
 import type { BasicOutputBuilderParams } from './basic-output-params';
 
 /**
- * Options for building an Alias Output
+ * Options for building an Alias Output.
  */
 export interface AliasOutputBuilderParams extends BasicOutputBuilderParams {
+    /**
+     * A unique ID for the new alias.
```

Comment:
```suggestion
     * Unique identifier of the alias, which is the BLAKE2b-160 hash of the Output ID that created it.
     * Unless its a newly created alias, then the id is zeroed.
```

Reply:
Should be `BLAKE2b-256` according to the TIP

Reply:
True, copied it from somewhere else, but there it was also wrong already (or rather outdated, at some point it was 160 also in the TIP)